src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -88,7 +94,23 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
     case GetAndUpdateNIHStatus(userInfo: UserInfo) =>
       getProfile(userInfo, true)
       // do NOT pipe the response back to the parent!
-  }
+
+    case SyncWhitelist =>
+      val (bucket, file) = (FireCloudConfig.Nih.whitelistBucket, FireCloudConfig.Nih.whitelistFile)
+      val whitelist = Source.fromInputStream(HttpGoogleServicesDAO.getBucketObjectAsInputStream(bucket, file)).getLines().toSet
```

Comment:
Blows up, probably.  I'll work on a fix.

What have decided as the correct behavior: ignore, ignore X times and then remove access to all, or remove access to all right away?